templates/list.jsx

```diff
@@ -53,17 +53,13 @@ export default function List({ _columns, _orderedList, _items, ...props }) {
                       role="heading"
                       aria-level={a11y.ariaLevel({ id: _id, level: 'componentItem', override: _ariaLevel ?? itemAriaLevel })}
                     >
-                      <div className="list-item__title-inner">
-                        {html(compile(title))}
-                      </div>
+                      <div className="list-item__title-inner" dangerouslySetInnerHTML={{ __html: title }} />
```

Comment:
Perhaps amend to `compile(title)` so that Handlebars expressions are still dealt with

Reply:
Thanks, @chris-steele .  I've made this change.  That said, I'm unsure how to test this.

In components.json, I've added `{{{_id}}}` to a list item's title and body, but it isn't actually displayed:

`"title": "The component ID is {{{_id}}}",`

Does the `compile()` React helper require parameters as a second argument?  

In the JSX, for instance, if I change the list title to:

`dangerouslySetInnerHTML={{ __html: compile(title, { props }) }}`

Then, in components.json, this would work:

`"title": "The component ID is {{{props._id}}}",`

Here's the helper:

```
/**
 * Handlebars compile integration
 * @param {string} name Handlebars template
 * @param {...any} args Template arguments
 */
export function compile(template, ...args) {
  const output = Handlebars.compile(template)(...args);
  return output;
};
```

Reply:
Ah, yes, you are correct, `props` needs to be passed.